library/src/main/kotlin/com/liferay/apio/consumer/ApioConsumer.kt

```diff
@@ -71,16 +142,9 @@ private fun request(url: HttpUrl,
 	val request = createRequest(httpUrl, credential)
 
 	val okHttpClient = OkHttpClient()
-	if (BuildConfig.DEBUG) {
-		IdlingResources.registerOkHttp(okHttpClient, httpUrl.toString())
-	}
 
 	val execute = okHttpClient.newCall(request).execute()
 
-	if (BuildConfig.DEBUG) {
```

Comment:
? :(

Reply:
No Android, no views, so no need for idling resources, right? 🤔